### ISSUE TITLE
model generation when hyphen's are in table name or column name

### ISSUE THREAD

**Issue by tempcke** [phalcon/phalcon-devtools#663]
using 2.1.0
I realize it seems stupid to put a - in a table name or column name but I did not create the schema :P

If a table name has a hyphen in it then the generated class name has a hyphen in it which is invalid.
same with column's containing a hyphen .. the resulting properties, getters and setter methods all end up with hyphen's in them.

**Comment by sergeyklay** [comment]
@tempcke Could you please provide a small script to reproduce?

**Comment by tempcke** [comment]
@sergeyklay 
First I made a mistake, I'm using 2.0.10 not 2.1.0 :)
## sql
```sql
# mysql
CREATE TABLE IF NOT EXISTS `foo-bar` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `some-col` varchar(20) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB;
INSERT INTO `foo-bar` (`id`, `some-col`) VALUES (1, 'a');
```
## All models
```
$ phalcon all-models --get-set
```
**Result:**
```php
# app/models/Foo-Bar.php
class Foo-bar extends \Phalcon\Mvc\Model {
    protected $some-col;

    /**
     * Method to set the value of field some-col
     */
    public function setSome-col($some-col)  {
        $this->some-col = $some-col;
        return $this;
    }

    /**
     * Returns the value of field some-col
     */
    public function getSome-col()   {
        return $this->some-col;
    }

    /**
     * Returns table name mapped in the model.
     */
    public function getSource()   {
        return 'foo-bar';
    }
}
```

## one model
```
$ phalcon model --name foo-bar --get-set
```
**Result:**
```php
class FooBar extends \Phalcon\Mvc\Model {
    protected $some-col;

    /**
     * Method to set the value of field some-col
     */
    public function setSome-col($some-col)   {
        $this->some-col = $some-col;
        return $this;
    }

    /**
     * Returns the value of field some-col
     */
    public function getSome-col()    {
        return $this->some-col;
    }

    public function initialize()    {
        $this->setSource("foo-bar");
    }

    public function getSource()    {
        return 'foo-bar';
    }
}
```
So when doing one model with `phalcon model` it removes the - from the filename and classname but leaves it in the property and method names.

**Comment by sergeyklay** [comment]
@tempcke  Thanks. I'll try to sort out as soon as I can

**Comment by tempcke** [comment]
Thanks, though this isn't a critical issue as it seems rare that people will have -'s in table and column names, and it is easy to fix the generated output, however I wanted to make you aware of it.

PS: Thank you so much for the work you and others are doing.  Since I started using Phalcon I absolutely love it!

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x branch`. Feel free to open new issue if the problem appears again. Thank you for contributing.
